.github/workflows/multidim-interop.yml

```diff
@@ -9,11 +9,15 @@ name: libp2p multidimensional interop test
 
 jobs:
   run-multidim-interop:
-    uses: "./.github/workflows/run-testplans.yml"
-    with:
-      dir: "multidim-interop"
-  run-multidim-interop-ng:
+    env:
+      TEST_PLAN_DIR: ${{ inputs.dir }}
+      AWS_BUCKET: ${{ vars.S3_BUCKET_NAME }}
+      AWS_REGION: us-east-1
```

Comment:
Why are we adjusting this? Have all repos migrated to the new action? I'd like to delete this first then (i.e. have smaller, more focused PRs.)

Reply:
go and rust have. This doesn't affect them anyways. This is only for this repo.